doc/glossary.rst

```diff
@@ -390,6 +390,12 @@
 	Teuthology
 		The collection of software that performs scripted tests on Ceph.
 
+        User
+                An individual or a system actor (for example, an application)
+                that uses Ceph clients to interact with the :term:`Ceph Storage
+                Cluster` daemons. See :ref:`User<rados-ops-user>` and
```

Comment:
Suggest s/daemons//

Depending on how one defines things, RGW e.g. isn't strictly a cluster daemon, it's a client.